src/main/java/com/vinyl/service/UserService.java

```diff
@@ -28,6 +34,14 @@ public User addUser(User user) {
 		user.setAddress("Defaul Address");
 		user.setRole(r);
 
+		Pattern p = Pattern.compile("^[A-Z][a-zA-Z]+$");
```

Comment:
create a String constant for the pattern "^[A-Z][a-zA-Z]+$"

Reply:
done